model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaModelCriteriaBuilder.java

```diff
@@ -144,4 +148,12 @@ protected Set<UUID> getUuidsForInOperator(Object[] values, SearchableModelField<
                     .filter(Objects::nonNull)
                     .collect(Collectors.toSet());
     }
+
+    protected byte[] sha256(String input) {
+        try {
+            return MessageDigest.getInstance("SHA-256").digest(input.getBytes());
```

Comment:
Are we certain this returns the same value as `sha256` function in Postgres?

Reply:
Yes, it returns `bytea`. But e.g. cocroach resturns `string`. Anyway it's now removed.